.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: '3.7'
+          python-version: '3.8'
```

Comment:
Update to 3.9, here and elsewhere

Reply:
Added 3.9 everywhere, as well as dropping some of the conditional checks for 3.7

Reply:
Maybe we should make a follow-up ticket/issue to thoroughly search the code to ensure the cleanup form 3.7 and 3.8 is complete.